package.json

```diff
@@ -117,5 +117,8 @@
     "packages": [
       "packages/**"
     ]
+  },
+  "dependencies": {
+    "@types/react-virtualized": "^9.21.4"
```

Comment:
i don't think this is needed in the root anymore since we've added it in react-virtualized-extension's package.json

Reply:
how would you suggest I address this? would it just be `yarn remove @types/react-virtualized`?

Reply:
I think so, yes, or you could remove the line from the package.json and re-run `yarn install`. If the build still passes without this dependency in here, yeah, I think remove it. But it's probably ok either way.

Reply:
You can delete the line. Likely no need to re-run `yarn install`, but better safe than sorry.